9c-main/chart/templates/remote-action-evaluator-headless.yaml

```diff
@@ -0,0 +1,245 @@
+apiVersion: apps/v1
```

Comment:
```suggestion
{{ if .Values.remoteActionEvaluatorHeadless.enabled }}
apiVersion: apps/v1
```

Reply:
I wrapped the resources with the `if-end` condition.

https://github.com/planetarium/9c-infra/compare/5a993f178cd4d204f9d5200a7c520bc2af58298e..13a68b069e244b3c73e4121d300cab84ac800e4a